app/controllers/api/v1/targets_controller.rb

```diff
@@ -16,12 +16,21 @@ def create
           render json: { errors: @target.errors }, status: :unprocessable_entity
         end
       end  
+
+      def destroy
+        target.destroy
```

Comment:
```suggestion
        target.destroy!
```
https://apidock.com/rails/ActiveRecord/Persistence/destroy%21

Reply:
Done!